android/lib/src/main/java/org/vosk/android/SpeechStreamService.java

```diff
@@ -126,16 +131,29 @@ public RecognizerThread(RecognitionListener listener) {
         public void run() {
 
             byte[] buffer = new byte[bufferSize];
+            ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
 
             while (!interrupted()
                     && ((timeoutSamples == NO_TIMEOUT) || (remainingSamples > 0))) {
                 try {
                     int nread = inputStream.read(buffer, 0, buffer.length);
+
+                    byte[] bdata = new byte[nread * 2];
```

Comment:
This is old code that can be removed - we are not outputting the resultBuffer, right?